bundles.rst

```diff
@@ -162,6 +163,80 @@ to be adjusted if needed:
             }
         }
 
+Developing a Reusable Bundle
+----------------------------
+
+Bundles are meant to be reusable pieces of code that live independently from
+any particular Symfony application. However, a bundle cannot run on its own: it
+must be registered inside an application to execute its code.
+
+This can be a bit challenging during development. When working on a bundle in
+its own repository, there's no Symfony application around it, so you need a way
+to test your changes inside a real application environment.
+
+There are two common approaches to do this, depending on whether your bundle has
+already been published or is still under development.
+
+Using a Local Path Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your bundle hasn't been published yet (for example, it's not available on
+Packagist), you can point Composer to your local bundle directory from any
+Symfony application you use for testing.
+
+Edit the ``composer.json`` file of your application and add this:
+
+.. code-block:: json
+
+    {
+        "repositories": [
+            {
+                "type": "path",
+                "url": "/path/to/your/AcmeBlogBundle"
```

Comment:
```suggestion
                "url": "path/to/your/AcmeBlogBundle"
```

An absolute path won’t necessarily work on another machine, CI, etc.

It looks like the path is relative to the `composer.json` file itself. https://getcomposer.org/doc/05-repositories.md#artifact

A note may clarify it.

---

Update: my bad, it makes sense in this context to use an absolute path since it’s only for local testing.